README.md

```diff
@@ -160,6 +160,9 @@ df = pd.DataFrame(
 To generate the report, run:
 ```python
 profile = ProfileReport(df, title="Pandas Profiling Report")
+
+# when dtypes are changed by `astype` function. Detect (new)dtypes from dataframe itself
+profile2 = ProfileReport(df, title="Pandas Profiling Report", infer_dtypes=False)
```

Comment:
For a user this comes to early in the documentation. I'd leave this one out (we already have this example below)

Reply:
sure.